src/components/veNft/formControls/StakingFormActionButton.tsx

```diff
@@ -31,7 +31,12 @@ const StakingFormActionButton = ({
     }
     if (!hasAdequateApproval) {
       return (
-        <Button block style={{ whiteSpace: 'pre' }} onClick={approve}>
+        <Button
+          block
+          style={{ whiteSpace: 'pre' }}
```

Comment:
**quesiton** do we need this?

Reply:
this was addressed in a downstream branch